src/pretix/plugins/sendmail/urls.py

```diff
@@ -13,4 +13,9 @@
         views.EmailHistoryView.as_view(),
         name='history',
     ),
+    url(
```

Comment:
Please use new way of defining URL (`path`). Other use of `url` should be updated some days.